.github/workflows/ci_cd.yml

```diff
@@ -42,19 +57,34 @@ jobs:
        with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
 
-  pr-title-style:
-    name: "Pull-request title style"
+  changelog:
```

Comment:
Why not put ``needs: pr-title-style`` for generating a proper changelog? We only execute it if the PR title is following the convention that way

Reply:
Good point!